package.json

```diff
@@ -13,22 +13,22 @@
     "./use-at-your-own-risk": "./lib/unsupported-api.js"
   },
   "scripts": {
+    "build:docs:update-links": "node tools/fetch-docs-links.js",
+    "build:release": "node Makefile.js generateRelease",
+    "build:release:alpha": "node Makefile.js generatePrerelease -- alpha",
+    "build:release:beta": "node Makefile.js generatePrerelease -- beta",
+    "build:release:publish": "node Makefile.js publishRelease",
+    "build:release:rc": "node Makefile.js generatePrerelease -- rc",
+    "build:site": "node Makefile.js gensite",
+    "build:webpack": "node Makefile.js webpack",
+    "lint": "node Makefile.js lint",
+    "lint:fix": "node Makefile.js lint -- fix",
+    "lint:docs:js": "node Makefile.js lintDocsJS",
+    "lint:docs:js:fix": "node Makefile.js lintDocsJS -- fix",
     "test": "node Makefile.js test",
     "test:cli": "mocha",
-    "lint": "node Makefile.js lint",
-    "lint:docsjs": "node Makefile.js lintDocsJS",
-    "fix": "node Makefile.js lint -- fix",
-    "fix:docsjs": "node Makefile.js lintDocsJS -- fix",
-    "fuzz": "node Makefile.js fuzz",
-    "generate-release": "node Makefile.js generateRelease",
-    "generate-alpharelease": "node Makefile.js generatePrerelease -- alpha",
-    "generate-betarelease": "node Makefile.js generatePrerelease -- beta",
-    "generate-rcrelease": "node Makefile.js generatePrerelease -- rc",
-    "publish-release": "node Makefile.js publishRelease",
```

Comment:
These scripts are used on Jenkins, so we need to decide if we'll make a change there or keep these names here.

Reply:
I don't know how complex / risky it is to update Jenkins. If you decide to use the new names, might want to do it like this. 

1. Add the new names (without removing the old names).  
2. Update Jenkins to use the new names.
3. A second commit that removes the old names.

Reply:
Let’s just leave these as-is right now. These aren’t just used within Jenkins, they are baked into eslint-release, which is used by all of our packages, making changing these script names more complex.

Reply:
Sorry, was too tired when I wrote this. The only place using these command is in Jenkins, and so long as we update Jenkins before merging this, I don’t see a problem.